src/prefect/deployments/deployments.py

```diff
@@ -671,7 +671,10 @@ async def upload_to_storage(
 
     @sync_compatible
     async def apply(
-        self, upload: bool = False, work_queue_concurrency: int = None
+        self,
```

Comment:
the changes in this file can be reverted - they were temporary when we were first getting wired up